modules/transactions/transactions.js

```diff
@@ -79,9 +79,11 @@ class Transactions {
             to = f.pkh;
         } else if (f = Helper.findKeyObj(config.contracts, to)) {
             to = f.pkh;
+        } else {
+            to = to;
         }
 
-        fees = fees || 1500;
+        fees = fees || 30000;
```

Comment:
make sure we're not using too much gas for transfer operation. 
Or use a standard value for mainnet else people will end up spending more gas unnecesserily.